---
layout: talk_reveal3
title: Real-time genomic surveillance of pathogen evolution and spread
venue: ETH Zurich
event: D-BSSE Seminar Series
author: Trevor Bedford
---

<!-- 45 minutes for talk, 10 minutes for questions -->

<!--
Outline
 1. Flu
 2. Ebola
 3. Zika
 4. nextstrain
-->

<!-- Color ramp: ["#5097BA", "#5BA6A7", "#69B091", "#79B77D", "#8BBB6A", "#9EBE5A", "#B2BD4D", "#C4B945", "#D4B13F", "#DFA43B"] -->

<section data-background="#000000">
	<h2 class="title">{{ page.title }}</h2>
	<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
</section>

<section data-background="#000000">
	<h3 class="title">Slides at bedford.io/talks/</h3>
</section>

<section data-background="#5097BA">
	<h2 class="title">Phylogenies</h2>
</section>

<section>
	<h3>Phylogenies describe history</h3>
	<img class="stretch" src="/images/talks/phylogenies_tudors.jpg">
</section>

<section>
	<h3>Phylogenies describe history</h3>
	<img class="stretch" src="/images/talks/phylogenies_haeckel.jpg">
	<div class="citation">
		Haeckel 1879
	</div>
</section>

<section>
	<h3>Phylogenies describe history</h3>
	<img class="stretch" src="/images/talks/phylogenies_pace.png">
	<div class="citation">
		<a href="http://www.sciencemag.org/content/276/5313/734.short">Pace 1997</a>
	</div>
</section>

<section>
	<h3>Phylogenies reveal process</h3>
	<img class="stretch" src="/images/talks/phylogenies_darwin.png">
	<div class="citation">
		Darwin 1859
	</div>
</section>

<section data-transition="slide-in none-out">
	<h3>Epidemic process</h3>
	<img class="stretch" src="/images/talks/infection_tree_1.png">
</section>

<section data-transition="none">
	<h3>Sample some individuals</h3>
	<img class="stretch" src="/images/talks/infection_tree_2.png">
</section>

<section data-transition="none">
	<h3>Sequence and determine phylogeny</h3>
	<img class="stretch" src="/images/talks/infection_tree_3b.png">
</section>

<section data-transition="none-in slide-out">
	<h3>Sequence and determine phylogeny</h3>
	<img class="stretch" src="/images/talks/infection_tree_4b.png">
</section>

<section>
	<h3>Localized Middle Eastern MERS-CoV phylogeny</h3>
	<img class="stretch" src="/images/talks/pathogen_phylogeny_mers.jpg">
	<div class="citation">
		<a href="http://www.thelancet.com/journals/lancet/article/PIIS0140-6736(13)61887-5/abstract">Cotten et al 2013</a>
	</div>
</section>

<section>
	<h3>Regional West African Ebola phylogeny</h3>
	<img class="stretch" src="/images/talks/pathogen_phylogeny_ebola.png">
	<div class="citation">
		<a href="http://bedford.io/papers/park-ebola-transmission/">Park et al 2015</a>
	</div>
</section>

<section>
	<h3>Global influenza phylogeny</h3>
	<img class="stretch" src="/images/talks/pathogen_phylogeny_flu.png">
	<div class="citation">
		<a href="http://bedford.io/papers/bedford-global-circulation/">Bedford et al 2015</a>
	</div>
</section>

<section>
	<h3>Applications of evolutionary analysis for <b>influenza vaccine strain selection</b> and <b>charting spread of Ebola and Zika</b></h3>
</section>

<section data-background="#5BA6A7">
	<h2 class="title">Influenza</h2>
</section>

<section>
	<h3>Influenza virion</h3>
	<img class="stretch" src="/images/talks/flu_virion.jpg">
</section>

<section>
	<h3>Population turnover (in H3N2) is extremely rapid</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_beast.png">
</section>

<section>
	<h3>Clades emerge, die out and take over</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_clades.png">
</section>

<section>
	<h3>Clades show rapid turnover</h3>
	<img class="stretch" data-src="/images/talks/prediction_combined_trajectories.png">
</section>

<section>
	<h3>Dynamics driven by antigenic drift</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree.png">
</section>

<section>
	<h3>Drift variants emerge and rapidly take over in the virus population</h3>
	<br>
	<h3>This causes the side effect of evading existing vaccine formulations</h3>
</section>

<section>
	<h3>Drift necessitates vaccine updates</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_vaccines.png">
</section>

<section>
	<h3>H3N2 vaccine updates occur every ~2 years</h3>
	<img class="stretch" data-src="/images/talks/flu_vaccine_updates.png">
</section>

<section>
	<h3>Timely surveillance and rapid analysis essential to vaccine strain selection</h3>
</section>

<section>
	<h1>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
	</h1>
	<p>Project to provide a real-time view of the evolving influenza population</p>
</section>

<section>
	<h2>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
	</h2>
	<p>Project to provide a real-time view of the evolving influenza population</p>
	<p>All in collaboration with <a href="https://neherlab.wordpress.com/">Richard Neher</a></p>
	<img class="stretch" data-src="/images/talks/neher.jpg">
</section>

<section>
	<h3>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
		pipeline
	</h3>
	<p>
	<ol>
		<li>Download all recent HA sequences from GISAID</li>
		<li>Filter to remove outliers</li>
		<li>Subsample across time and space</li>
		<li>Align sequences</li>
		<li>Build tree</li>
		<li>Estimate clade frequencies</li>
		<li>Infer antigenic phenotypes</li>
		<li>Export for visualization</li>
	</ol>
</section>

<section data-background="#000000">
	<h1 style="font-weight: 500;">
		<a href="http://nextflu.org" target="_blank">
			<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span><span class="text-light-gray">.org</span>
		</a>
	</h1>
</section>

<!-- Live demo of flu -->

<!-- <section data-background="#79B77D">
	<h2 class="title">Antigenic evolution</h2>
</section>

<section>
	<h3>Influenza hemagglutination inhibition (HI) assay</h3>
	<img class="stretch" src="/images/talks/hi_assay.png">
</section>

<section>
	<h3>HI measures cross-reactivity across viruses</h3>
	<img class="stretch" src="/images/talks/hi_cross_reactivity.jpg">
</section>

<section>
	<h3>Data in the form of table of maximum inhibitory titers</h3>
	<img class="stretch" src="/images/talks/hi_table.png">
	<div class="citation">
		<a href="http://www.crick.ac.uk/research/worldwide-influenza-centre/annual-and-interim-reports/">NIMR 2010</a>
	</div>
</section>

<section>
	<h3>Model HI titers by estimating branch effects</h3>
	<img class="stretch" src="/images/talks/titer_model_with_table.png">
	<div class="citation">
		<a href="http://bedford.io/papers/neher-titers/">Neher et al 2016</a>
	</div>
</section>

<section>
	<h3>Model is highly predictive of missing titer values</h3>
	<img class="stretch" src="/images/talks/titers_validation.png">
	<div class="citation">
		<a href="http://bedford.io/papers/neher-titers/">Neher et al 2016</a>
	</div>
</section>

<section>
	<h3>Recent HI data from WHO CC London annual and interim reports</h3>
</section>

<section>
	<h3>Up-to-date analysis at:</h3>
	<h2>
		<a href="http://nextflu.org">
			<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span><span class="text-gray">.org</span>
		</a>
	</h2>
</section> -->

<section data-background="#69B091">
	<h2 class="title">Forecasting</h2>
</section>

<section>
	<h3><p><q>The future is here, it's just not evenly distributed yet</q> &mdash; William Gibson</p></h3>
</section>

<section>
	<h3>USA music industry, 2011 dollars per capita</h3>
	<img class="stretch" src="/images/talks/music_industry.jpg">
</section>

<section>
	<h3>Influenza population turnover</h3>
	<img class="stretch" src="/images/talks/flu_turnover.png">
</section>

<section>
	<h3>Vaccine strain selection timeline</h3>
	<img class="stretch" src="/images/talks/flu_vaccine_schedule.jpg">
</section>

<section>
	<h3>Seek to explain change in clade frequencies over 1 year</h3>
	<img class="stretch" src="/images/talks/prediction_clade_frequencies.png">
</section>

<section>
	<h3>Fitness models can project clade frequencies</h3>
	<br>
	<p>Clade frequencies $X$ derive from the fitnesses $f$ and frequencies $x$ of constituent viruses, such that</p>
	<p>$$\hat{X}_v(t+\Delta t) = \sum_{i:v} x_i(t) \, \mathrm{exp}(f_i \, \Delta t)$$</p>
	<p>This captures clonal interference between competing lineages</p>
	<img class="stretch" src="/images/talks/clonal_interferance.jpg">
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig 2014</a>
	</div>
</section>

<section>
	<h3>Predictive fitness models</h3>
	<br>
	<p>A simple predictive model estimates the fitness $f$ of virus $i$ as</p>
	<p>$$\hat{f}_i = \beta^\mathrm{ep} \, f_i^\mathrm{ep} + \beta^\mathrm{ne} \, f_i^\mathrm{ne}$$</p>
	<p>where $f_i^\mathrm{ep}$ measures cross-immunity via substitutions at epitope sites and $f_i^\mathrm{ep}$ measures mutational load via substitutions at non-epitope sites</p>
	<img class="stretch" src="/images/talks/prediction_ll.jpg">
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig 2014</a>
	</div>
</section>

<section>
	<h3>We implement a similar model based on two predictors</h3>
	<br>
	<ol>
		<li class="larger">Clade frequency change</li>
		<li class="larger">Antigenic advancement</li>
	</ol>
</section>

<section>
	<h3>Project frequencies forward, <br> growing clades have high fitness</h3>
	<img class="stretch" src="/images/talks/prediction_frequency_trajectories.png">
</section>

<section>
	<h3>Calculate antigenic difference from ancestor via serological data</h3>
	<img class="stretch" src="/images/talks/prediction_antigenic_drift.png">
</section>

<section>
	<h3>Fitness model parameterization</h3>
	<br>
	<p>Our predictive model estimates the fitness $f$ of virus $i$ as</p>
	<br>
	<p>$$\hat{f}_i = \beta^\mathrm{freq} \, f_i^\mathrm{freq} + \beta^\mathrm{HI} \, f_i^\mathrm{HI}$$</p>
	<br>
	<p>We learn coefficients and validate model based on previous 15 H3N2 seasons</p>
</section>

<section>
	<h3>Clade growth rate is well correlated (<i>&rho;</i> = 0.66)</h3>
	<img class="stretch" src="/images/talks/prediction_validation_growth_rates.png">
</section>

<section>
	<h3>Growth vs decline correct in 84% of cases</h3>
	<img class="stretch" src="/images/talks/prediction_validation_growth_rates_window.png">
</section>

<section>
	<h3>Trajectories show more detailed congruence</h3>
	<img class="stretch" src="/images/talks/prediction_trajectories_grid.png">
</section>

<section>
	<h3>Real-time analyses are <i>actionable</i> and may inform influenza vaccine strain selection</h3>
</section>

<section data-background="#79B77D">
	<h2 class="title">Ebola</h2>
</section>

<section>
	<h2>Tracking geographic spread of the Ebola epidemic</h2>
	<p class="smaller"><i>
	with Gytis Dudas, Luiz Carvalho, Marc Suchard, Philippe Lemey, Andrew Rambaut <br> and many others
	</i></p>
	<div class="citation">
		<a href="/papers/dudas-ebola-epidemic-spread/">Dudas et al. 2016. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>Sequencing of 1610 Ebola virus genomes collected during the 2013-2016 West African epidemic</h3>
	<img class="stretch" data-src="/images/talks/ebola_mig_sampling.png">
	<div class="citation">
		<a href="/papers/dudas-ebola-epidemic-spread/">Dudas et al 2016</a>
	</div>
</section>

<section>
	<h3>Phylogenetic reconstruction of evolution and spread</h3>
	<img class="stretch" data-src="/images/talks/ebola_mig_tree.png">
	<div class="citation">
		<a href="/papers/dudas-ebola-epidemic-spread/">Dudas et al 2016</a>
	</div>
</section>

<section>
	<h3>Tracking migration events</h3>
	<img class="stretch" data-src="/images/talks/ebola_mig_geo_tree.png">
	<div class="citation">
		<a href="/papers/dudas-ebola-epidemic-spread/">Dudas et al 2016</a>
	</div>
</section>

<!-- possible to drop for time -->
<section>
	<h3>Factors influencing migration rates</h3>
	<img class="stretch" data-src="/images/talks/ebola_mig_glm_factors.png">
	<div class="citation">
		<a href="/papers/dudas-ebola-epidemic-spread/">Dudas et al 2016</a>
	</div>
</section>

<!-- possible to for time -->
<section>
	<h3>Effect of borders on migration rates</h3>
	<img class="stretch" data-src="/images/talks/ebola_mig_border_effects.png">
	<div class="citation">
		<a href="/papers/dudas-ebola-epidemic-spread/">Dudas et al 2016</a>
	</div>
</section>

<section>
	<h3>Spatial structure at the country level</h3>
	<img class="stretch" data-src="/images/talks/ebola_mig_exploded_tree.png">
	<div class="citation">
		<a href="/papers/dudas-ebola-epidemic-spread/">Dudas et al 2016</a>
	</div>
</section>

<section>
	<h3>Substantial mixing at the regional level</h3>
	<img class="stretch" data-src="/images/talks/ebola_mig_exploded_tree_regions_sierra_leone.png">
	<div class="citation">
		<a href="/papers/dudas-ebola-epidemic-spread/">Dudas et al 2016</a>
	</div>
</section>

<section>
	<h3>Regional outbreaks due to multiple introductions</h3>
	<img class="stretch" data-src="/images/talks/ebola_mig_clusters_sierra_leone.png">
	<div class="citation">
		<a href="/papers/dudas-ebola-epidemic-spread/">Dudas et al 2016</a>
	</div>
</section>

<section>
	<h3>Each introduction results in a minor outbreak</h3>
	<img class="stretch" data-src="/images/talks/ebola_mig_cluster_distributions.png">
	<div class="citation">
		<a href="/papers/dudas-ebola-epidemic-spread/">Dudas et al 2016</a>
	</div>
</section>

<!-- ffmpeg -i Untitled.mov -filter_complex "setpts=0.75*PTS" -an -vcodec libx264 ebola_mig_movie.mp4 -->
<section data-autoplay data-background-video="/images/talks/ebola_mig_movie.mp4">
</section>

<section>
	<h3>Ebola spread in West Africa followed a gravity model with moderate slowing by international borders,
		in which spread is driven by short-lived migratory clusters</h3>
</section>

<section data-background="#8BBB6A">
	<h2 class="title">Zika</h2>
</section>

<section>
	<h3>Zika's arrival and spread in the Americas</h3>
	<img class="stretch" src="/images/talks/zika_who_map.png">
	<div class="citation">
		<a href="http://www.who.int/emergencies/zika-virus/situation-report/17-november-2016/en/">WHO 2016</a>
	</div>
</section>

<section>
	<h2>Tracking origins of the Zika epidemic</h2>
	<p class="smaller"><i>
	with Nuno Faria, Nick Loman, Oli Pybus, Luiz Alcantara, Ester Sabino, Josh Quick, Allison Black,
	Ingra Morales, Julien Thézé, Marcio Nunes, Jacqueline de Jesus, Marta Giovanetti, Moritz Kraemer,
	Sarah Hill and many others
	</i></p>
	<div class="citation">
		<a href="/papers/faria-zika-in-brazil/">Faria et al. 2017. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>Road trip through northeast Brazil to collect samples and sequence</h3>
	<img class="stretch" src="/images/talks/zika_zibra_roadtrip_samples.jpg">
</section>

<section data-background="/images/talks/zika_zibra_bus.jpg">
</section>

<section data-background="/images/talks/zika_zibra_trevor_in_lab.jpg">
</section>

<section data-background="/images/talks/zika_zibra_minion.jpg">
</section>

<section>
	<h3>Case reports and diagnostics suggest initiation in northeast Brazil</h3>
	<img class="stretch" src="/images/talks/zika_zibra_epi_timeseries.png">
	<div class="citation">
		<a href="/papers/faria-zika-in-brazil/">Faria et al. 2017</a>
	</div>
</section>

<section>
	<h3>Sequencing shows accumulating genetic diversity</h3>
	<img class="stretch" src="/images/talks/zika_zibra_sequencing.png">
	<div class="citation">
		<a href="/papers/faria-zika-in-brazil/">Faria et al. 2017</a>
	</div>
</section>

<section>
	<h3>Phylogeny infers an origin in northeast Brazil</h3>
	<img class="stretch" src="/images/talks/zika_zibra_tree.png">
	<div class="citation">
		<a href="/papers/faria-zika-in-brazil/">Faria et al. 2017</a>
	</div>
</section>

<section>
	<h2>Local spread of Zika in Florida</h2>
	<p class="smaller"><i>
	with Kristian Andersen, Nathan Grubaugh, Jason Ladner, Gustavo Palacios, Sharon Isern, Oli Pybus,
	Moritz Kraemer, Gytis Dudas, Amanda Tan, Karthik Gangavarapu, Michael Wiley, Stephen White,
	Julien Thézé, Scott Michael, Leah Gillis, Pardis Sabeti, and many others
	</i></p>
	<div class="citation">
		<a href="/papers/grubaugh-zika-in-florida/">Grubaugh et al. 2017. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>Outbreak of locally-acquired infections focused in Miami-Dade county</h3>
	<img class="stretch" src="/images/talks/zika_florida_epi_timeseries.png">
	<div class="citation">
		<a href="/papers/grubaugh-zika-in-florida/">Grubaugh et al. 2017</a>
	</div>
</section>

<section>
	<h3>Phylogeny shows a surprising degree of clustering</h3>
	<img class="stretch" src="/images/talks/zika_florida_tree.png">
	<div class="citation">
		<a href="/papers/grubaugh-zika-in-florida/">Grubaugh et al. 2017</a>
	</div>
</section>

<section>
	<h3>Clustering suggests fewer, longer transmission chains and higher <i>R</i><sub>0</sub></h3>
	<img class="stretch" src="/images/talks/zika_florida_r0.png">
	<div class="citation">
		<a href="/papers/grubaugh-zika-in-florida/">Grubaugh et al. 2017</a>
	</div>
</section>

<section>
	<h3>Extrapolate <i>R</i><sub>0</sub> to predict introduction counts driving outbreak</h3>
	<img class="stretch" src="/images/talks/zika_florida_introduction_count.png">
	<div class="citation">
		<a href="/papers/grubaugh-zika-in-florida/">Grubaugh et al. 2017</a>
	</div>
</section>

<section>
	<h3>Flow of infected travelers greatest from Caribbean</h3>
	<img class="stretch" src="/images/talks/zika_florida_travel_patterns.png">
	<div class="citation">
		<a href="/papers/grubaugh-zika-in-florida/">Grubaugh et al. 2017</a>
	</div>
</section>

<section>
	<h3>Southern Florida has high potential for <i>Aedes</i> borne outbreaks</h3>
	<img class="stretch" src="/images/talks/zika_florida_suitability.png">
	<div class="citation">
		<a href="/papers/grubaugh-zika-in-florida/">Grubaugh et al. 2017</a>
	</div>
</section>

<section data-background="#9EBE5A">
	<h3 class="title">Important analyses, let's make them more rapid and more automated</h3>
</section>

<section style="text-align: left;">
	<h3>Key challenges</h3>
	<p>
	<ul>
		<li>Timely analysis and sharing of results critical</li>
		<li>Dissemination must be scalable</li>
		<li>Integrate many data sources</li>
		<li>Results must be easily interpretable and queryable</li>
	</ul>
	</p>
</section>

<section>
	<h1>
		<span class="c1s">n</span><span class="c2s">e</span><span class="c3s">x</span><span class="c4s">t</span><span class="c5s">s</span><span class="c6s">t</span><span class="c7s">r</span><span class="c8s">a</span><span class="c9s">i</span><span class="c10s">n</span>
	</h1>
	<p>Project to conduct real-time molecular epidemiology and evolutionary analysis of emerging epidemics</p>
	<br>
	<p>
		<img width=50 class="img-avatar" src="/images/team/richard-neher.jpg"> Richard Neher,
		<img width=50 class="img-avatar" src="/images/team/trevor-bedford.jpg"> Trevor Bedford,
		<img width=50 class="img-avatar" src="/images/team/colin-megill.jpg"> Colin Megill,
		<br>
		<img width=50 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,
		<img width=50 class="img-avatar" src="/images/team/charlton-callender.jpg"> Charlton Callender,
		<img width=50 class="img-avatar" src="/images/team/sidney-bell.jpg"> Sidney Bell,
		<br>
		<img width=50 class="img-avatar" src="/images/team/barney-potter.jpg"> Barney Potter,
		<img width=50 class="img-avatar" src="/images/team/sarah-murata.jpg"> Sarah Murata,
	</p>
</section>

<section>
	<h3>Nextstrain architecture</h3>
	<img class="stretch" src="/images/talks/nextstrain_components.png">
</section>

<section data-background="#000000">
	<h1 style="font-weight: 500;">
		<a href="http://nextstrain.org/" target="_blank">
			<span class="c1s">n</span><span class="c2s">e</span><span class="c3s">x</span><span class="c4s">t</span><span class="c5s">s</span><span class="c6s">t</span><span class="c7s">r</span><span class="c8s">a</span><span class="c9s">i</span><span class="c10s">n</span><span class="text-light-gray">.org</span>
		</a>
	</h1>
</section>

<!-- Live demo of Ebola and Zika -->

<section data-background="/images/talks/nextstrain_in_sierra_leone.jpg">
	<h3 class="title">Rapid on-the-ground sequencing by Ian Goodfellow, Matt Cotten and colleagues</h3>
	<br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p>
</section>

<section data-background="#B2BD4D">
	<h3 class="title">Desired analytics are pathogen specific and tied to response measures</h3>
</section>

<section>
	<h3>Acknowledgements</h3>

	<p class="smaller"><b>Influenza</b>: WHO Global Influenza Surveillance Network, Worldwide Influenza Centre at the Francis Crick Institute, Richard Neher, Colin Russell, Boris Shraiman</p>

	<p class="smaller"><b>Ebola</b>: data producers, Gytis Dudas, Andrew Rambaut, Luiz Carvalho, Philippe Lemey,
	Marc Suchard, Andrew Tatem, Nick Loman, Ian Goodfellow, Matt Cotten, Paul Kellam, Kristian Andersen,
	Pardis Sabeti, many others</p>

	<p class="smaller"><b>Zika</b>: data producers, Nick Loman, Nuno Faria,	Oliver Pybus, Josh Quick,
	Allison Black, Kristian Andersen, Nathan Grubaugh, Gytis Dudas, many others</p>

	<p class="smaller"><b>Nextstrain</b>: Richard Neher, Colin Megill, James Hadfield, Charlton Callender,
	Sarah Murata, Sidney Bell, Barney Potter</p>

	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="15%" data-src="/images/talks/logo_fred_hutch_vertical.png">
		&nbsp; &nbsp;
		<img width="11%" data-src="/images/talks/logo_pew.png">
		&nbsp; &nbsp;
		<img width="10%" data-src="/images/talks/logo_hhmi.jpg">
		&nbsp; &nbsp;
		<img width="12%" data-src="/images/talks/logo_wellcome_trust.jpg">
		&nbsp; &nbsp;
		<img width="10%" data-src="/images/talks/logo_nih.jpg">
		&nbsp; &nbsp;
		<img width="15%" data-src="/images/talks/logo_osp.jpg">
	</div>
</section>

<section data-background="#000000">
</section>
