---
layout: talk_reveal3
title: Real-time forecasting of influenza virus evolution
venue: Tokyo, Japan
event: CREST International Symposium on Big Data Applications
author: Trevor Bedford
---

<!-- Color ramp: ["#5097BA", "#5BA6A7", "#69B091", "#79B77D", "#8BBB6A", "#9EBE5A", "#B2BD4D", "#C4B945", "#D4B13F", "#DFA43B"] -->

<!-- New color ramp: #BD698B, #69BBBD, #A6BD69, #697BBD, #BD9969 -->

<section data-background="#000000">
	<h1 class="title">{{ page.title }}</h1>
	<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
</section>

<section>
	<img class="stretch" data-src="/images/talks/forecasting_weather.png">
	<div class="citation">
		<a href="https://www.wunderground.com/cgi-bin/findweather/getForecast?query=chicago">Weather Underground</a>
	</div>
</section>

<section data-background="#000000">
	<h2 class="title">Want to forecast the make up of the future flu population from the population that exists today</h2>
</section>

<section>
	<h3>Population turnover (in H3N2) is extremely rapid</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_beast.png">
</section>

<section>
	<h3>Clades emerge, die out and take over</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_clades.png">
</section>

<section>
	<h3>Clades show rapid turnover</h3>
	<img class="stretch" data-src="/images/talks/prediction_combined_trajectories.png">
</section>

<section>
	<h3>Dynamics driven by antigenic drift</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree.png">
</section>

<section data-background="#000000">
	<h2 class="title">Drift variants emerge and rapidly take over in the virus population</h2>
</section>

<section>
	<h3>Drift necessitates vaccine updates</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_vaccines.png">
</section>

<section>
	<h3>H3N2 vaccine updates occur every ~2 years</h3>
	<img class="stretch" data-src="/images/talks/flu_vaccine_updates.png">
</section>

<section data-background="#000000">
	<h2 class="title">Timely surveillance and rapid analysis essential to vaccine strain selection</h2>
</section>

<section>
	<h1>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
	</h1>
	<p>Project to provide a real-time view of the evolving influenza population</p>
</section>

<section>
	<h2>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
	</h2>
	<p>Project to provide a real-time view of the evolving influenza population</p>
	<p>All in collaboration with <a href="https://neherlab.wordpress.com/">Richard Neher</a></p>
	<img class="stretch" data-src="/images/talks/neher.jpg">
</section>

<section>
	<h3>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
		pipeline
	</h3>
	<p>
	<ol>
		<li>Download all recent HA sequences from GISAID</li>
		<li>Filter to remove outliers</li>
		<li>Subsample across time and space</li>
		<li>Align sequences</li>
		<li>Build tree</li>
		<li>Estimate clade frequencies</li>
		<li>Infer antigenic phenotypes</li>
		<li>Export for visualization</li>
	</ol>
</section>

<section>

	<section>
		<h3>Up-to-date analysis publicly available at:</h3>
		<h2>
			<a href="http://nextflu.org">
				<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span><span class="text-gray">.org</span>
			</a>
		</h2>
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_1.png">
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_2.png">
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_3.png">
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_4.png">
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_5.png">
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_6.png">
	</section>

</section>

<!-- Demo focusing on the emergence and spread of 3c2a and 3c3a viruses -->

<section data-background="#69BBBD">
	<h1 class="title">Antigenic analysis</h1>
</section>

<section>
	<h3>Influenza hemagglutination inhibition (HI) assay</h3>
	<img class="stretch" data-src="/images/talks/hi_assay.png">
</section>

<section>
	<h3>HI measures cross-reactivity across viruses</h3>
	<img class="stretch" data-src="/images/talks/hi_cross_reactivity.jpg">
</section>

<section>
	<h3>Data in the form of table of maximum inhibitory titers</h3>
	<img class="stretch" data-src="/images/talks/hi_table.png">
	<div class="citation">
		<a href="http://www.crick.ac.uk/research/worldwide-influenza-centre/annual-and-interim-reports/">NIMR 2010</a>
	</div>
</section>

<section>
	<div class="left" style="float:left;width:63%">
		<h3>Antigenic cartography compresses HI measurements into an interpretable diagram</h3>
	</div>
	<div class="right" style="float:right;width:33%">
		<img data-src="/images/talks/hi_smith_map.jpg">
	</div>
	<div class="citation">
		<a href="http://www.sciencemag.org/content/305/5682/371.abstract">Smith et al 2004</a>
	</div>
</section>

<section>
	<h3>Instead of a geometric model, we sought a phylogenetic model of HI titer data</h3>
	<img class="stretch" data-src="/images/talks/titers_tree_example.png">
	<div class="citation">
		<a href="http://bedford.io/papers/neher-titers/">Neher et al 2016</a>
	</div>
</section>

<section>
	<h3>Identify phylogeny branches associated with drops in HI titer</h3>
	<img class="stretch" data-src="/images/talks/titer_model_with_table.png">
	<div class="citation">
		<a href="http://bedford.io/papers/neher-titers/">Neher et al 2016</a>
	</div>
</section>

<section>
	<h3>Model can be used to interpolate across tree and predict phenotype of untested viruses</h3>
	<img class="stretch" data-src="/images/talks/titers_tree_model.png">
	<div class="citation">
		<a href="http://bedford.io/papers/neher-titers/">Neher et al 2016</a>
	</div>
</section>

<section>
	<h3>Model is highly predictive of missing titer values</h3>
	<img class="stretch" data-src="/images/talks/titers_validation.png">
	<div class="citation">
		<a href="http://bedford.io/papers/neher-titers/">Neher et al 2016</a>
	</div>
</section>

<section data-background="#000000">
	<h2 class="title">Incorporate HI data from US Centers for Disease Control and Prevention</h2>
</section>

<section>

	<section>
		<h3>Up-to-date analysis at:</h3>
		<h2>
			<a href="http://nextflu.org">
				<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span><span class="text-gray">.org</span>
			</a>
		</h2>
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_7.png">
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_8.png">
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_9.png">
	</section>

</section>

<!-- Demo of titer tool using H3N2 / 12y. Look at measurements and antigenic clusters surrounding vaccine strains. -->

<section data-background="#A6BD69">
	<h1 class="title">Forecasting</h1>
</section>

<section data-background="#000000">
	<h2 class="title"><p><q>"The future is here, it's just not evenly distributed yet"</q> <br> &mdash; William Gibson</p></h2>
</section>

<section>
	<h3>USA music industry, 2011 dollars per capita</h3>
	<img class="stretch" data-src="/images/talks/music_industry.jpg">
</section>

<section>
	<h3>Influenza population turnover</h3>
	<img class="stretch" data-src="/images/talks/flu_turnover.png">
</section>

<section>
	<h3>Vaccine strain selection timeline</h3>
	<img class="stretch" data-src="/images/talks/flu_vaccine_schedule.jpg">
</section>

<section>
	<h3>Seek to explain change in clade frequencies over 1 year</h3>
	<img class="stretch" data-src="/images/talks/prediction_clade_frequencies.png">
</section>

<section>
	<h3>Fitness models can project clade frequencies</h3>
	<br>
	<p>Clade frequencies $X$ derive from the fitnesses $f$ and frequencies $x$ of constituent viruses, such that</p>
	<p>$$\hat{X}_v(t+\Delta t) = \sum_{i:v} x_i(t) \, \mathrm{exp}(f_i \, \Delta t)$$</p>
	<p>This captures clonal interference between competing lineages</p>
	<img class="stretch" data-src="/images/talks/clonal_interferance.jpg">
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig 2014</a>
	</div>
</section>

<section data-background="#000000">
	<h2 class="title">The question of forecasting becomes: how do we accurately estimate fitnesses of circulating viruses?</h2>
</section>

<section>
	<h3>Fortunately, there's lots of training data and previously successful strains have had:</h3>
	<br>
	<ol>
		<li class="larger">Amino acid changes at epitope sites</li>
		<li class="larger">Antigenic novelty based on HI</li>
		<li class="larger">Rapid phylogenetic growth</li>
	</ol>
</section>

<section>
	<h3>Predictor: calculate HI drop from ancestor, <br> drifted clades have high fitness</h3>
	<img class="stretch" data-src="/images/talks/prediction_antigenic_drift.png">
</section>

<section>
	<h3>Predictor: project frequencies forward, <br> growing clades have high fitness</h3>
	<img class="stretch" data-src="/images/talks/prediction_frequency_trajectories.png">
</section>

<section>
	<h3>We predict fitness based on a simple formula</h3>
	<br>
	<p>where the fitness $f$ of virus $i$ is estimated as</p>
	<p>$$\hat{f}_i = \beta^\mathrm{HI} \, f_i^\mathrm{HI} + \beta^\mathrm{freq} \, f_i^\mathrm{freq}$$</p>
	<p>where $f_i^\mathrm{HI}$ measures antigenic drift via HI and $f_i^\mathrm{freq}$ measures clade growth/decline</p>
	<p>We learn coefficients and validate model based on previous 15 H3N2 seasons</p>
</section>

<section>
	<h3>Clade growth rate is well predicted (<i>&rho;</i> = 0.66)</h3>
	<img class="stretch" data-src="/images/talks/prediction_validation_growth_rates.png">
</section>

<section>
	<h3>Growth vs decline correct in 84% of cases</h3>
	<img class="stretch" data-src="/images/talks/prediction_validation_growth_rates_window.png">
</section>

<section data-transition="slide-in zoom-out" data-transition-speed="slow">
	<h3>Trajectories show more detailed congruence</h3>
	<img class="stretch" data-src="/images/talks/prediction_trajectories_grid.png">
</section>

<section data-transition="zoom-in slide-out">
	<h3>Trajectories show more detailed congruence</h3>
	<img class="stretch" data-src="/images/talks/prediction_trajectories_zoom.png">
</section>

<section>
	<h3>When does the forecast fail?</h3>
	<br>
	<p>Emerging clades are difficult to forecast: little antigenic data and little evidence of "past performance"</p>
	<p>Models work well for clades at >10%, but less well for clades <5%</p>
</section>

<section>
	<h3>New mutations difficult</h3>
	<br>
	<p>Models can project forward from circulating strains, but cannot foresee the appearance of new mutations</p>
	<p>Intrinsically limits the timescale of forecasting to ~1 year</p>
</section>

<section>
	<h3>Model is only as good as the data</h3>
	<br>
	<p>Requires rapid shipping of samples, rapid sequencing and rapid antigenic characterization</p>
</section>

<section data-background="#697BBD">
	<h1 class="title">Current situation</h1>
</section>

<section>
	<h3>Further improvements to predictive modeling</h3>
	<br>
	<ol>
		<li>Extend to other seasonal viruses</li>
		<li>Forecast NA evolution</li>
		<li>Integrate neutralization (FRA) assay data</li>
		<li>Model effects of egg adaptation</li>
		<li>Incorporate an explicit geographic model</li>
	</ol>
</section>

<section data-background="#000000">
	<h2 class="title">Real-time analyses are <i>actionable</i> and may inform influenza vaccine strain selection</h2>
</section>

<section data-background="#000000">
	<h2 class="title">More generally real-time analyses may be useful for other viruses</h2>
</section>

<section data-background="#BD9969">
	<h1 class="title">Zika</h1>
</section>

<section data-background="#E1E1E1">
	<h3>Zika's arrival and spread in the Americas</h3>
	<img class="stretch" src="/images/talks/zika_who_map.png">
	<div class="citation">
		<a href="http://www.who.int/emergencies/zika-virus/situation-report/17-november-2016/en/">WHO 2016</a>
	</div>
</section>

<section>
	<h3>Establishment and cryptic transmission of Zika virus in Brazil and the Americas</h3>
	<p class="smaller"><i>
	with Nuno Faria, Nick Loman, Oli Pybus, Luiz Alcantara, Ester Sabino, Josh Quick, <br>
	<img width=40 class="img-avatar" src="/images/team/allison-black.jpg"> Alli Black,
	Ingra Morales, Julien Thézé, Marcio Nunes, Jacqueline de Jesus, <br>
	Marta Giovanetti, Moritz Kraemer, Sarah Hill and many others
	</i></p>
	<div class="citation">
		<a href="/papers/faria-zika-in-brazil/">Faria et al. 2017. Nature.</a>
	</div>
</section>

<section data-background="#DCDCDC">
	<img class="stretch" src="/images/talks/giphy_road_trip.gif">
</section>

<section>
	<h3>Road trip through northeast Brazil to collect samples and sequence</h3>
	<img class="stretch" src="/images/talks/zika_zibra_roadtrip_samples.jpg">
	<div class="citation">
		<a href="/papers/faria-zika-in-brazil/">Faria et al. 2017. Nature.</a>
	</div>
</section>

<section data-background="/images/talks/zika_zibra_bus.jpg">
</section>

<section data-background="/images/talks/zika_zibra_trevor_in_lab.jpg">
</section>

<section data-background="/images/talks/zika_zibra_minion.jpg">
</section>

<section>
	<h3>Case reports and diagnostics suggest initiation in northeast Brazil</h3>
	<img class="stretch" src="/images/talks/zika_zibra_epi_timeseries.png">
	<div class="citation">
		<a href="/papers/faria-zika-in-brazil/">Faria et al. 2017. Nature.</a>
	</div>
</section>

<section>
	<h3>Phylogeny infers an origin in northeast Brazil</h3>
	<img class="stretch" src="/images/talks/zika_zibra_tree.png">
	<div class="citation">
		<a href="/papers/faria-zika-in-brazil/">Faria et al. 2017. Nature.</a>
	</div>
</section>

<section data-background="#000000">
	<h2 class="title">Important analyses, let's make them more rapid and more automated</h2>
</section>

<section>
	<h1>
		<span class="c1s">n</span><span class="c2s">e</span><span class="c3s">x</span><span class="c4s">t</span><span class="c5s">s</span><span class="c6s">t</span><span class="c7s">r</span><span class="c8s">a</span><span class="c9s">i</span><span class="c10s">n</span>
	</h1>
	<p>Project to conduct real-time molecular epidemiology and evolutionary analysis of emerging epidemics</p>
	<br>
	<p class="smaller"><i>
		with
		<img width=40 class="img-avatar" src="/images/team/richard-neher.jpg"> Richard Neher,
		<img width=40 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,
		<img width=40 class="img-avatar" src="/images/team/colin-megill.jpg"> Colin Megill,
		<br>
		<img width=40 class="img-avatar" src="/images/team/sidney-bell.jpg"> Sidney Bell,
		<img width=40 class="img-avatar" src="/images/team/charlton-callender.jpg"> Charlton Callender,
		<img width=40 class="img-avatar" src="/images/team/barney-potter.jpg"> Barney Potter,
		<br>
		and <img width=40 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston
	</i></p>
</section>

<section>
	<h3>Nextstrain architecture</h3>
	<p>All code open source at <a class="off" href="https://github.com/nextstrain">github.com/nextstrain</a></p>
	<img class="stretch" src="/images/talks/nextstrain_components.png">
</section>

<section data-background="#000000">
	<h1 style="font-weight: 500;">
		<a href="http://nextstrain.org/" target="_blank">
			<span class="c1s">n</span><span class="c2s">e</span><span class="c3s">x</span><span class="c4s">t</span><span class="c5s">s</span><span class="c6s">t</span><span class="c7s">r</span><span class="c8s">a</span><span class="c9s">i</span><span class="c10s">n</span><span class="text-light-gray">.org</span>
		</a>
	</h1>
</section>

<!-- Live demo of Zika -->

<section>
	<img class="stretch" src="/images/talks/gates_nextstrain_tweet.png">
</section>

<section data-background="/images/talks/nextstrain_in_sierra_leone.jpg">
	<h3 class="title">Rapid on-the-ground sequencing by Ian Goodfellow, Matt Cotten and colleagues</h3>
	<br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p>
</section>

<section data-background="#000000">
	<h2 class="title">Build out pipelines for different pathogens,
		improve databasing and lower
		bioinformatics bar</h2>
</section>

<section>
	<h3>Acknowledgements</h3>

	<p class="tiny"><b>Bedford Lab</b>:
		<img width=40 class="img-avatar" src="/images/team/allison-black.jpg"> Alli Black,
		<img width=40 class="img-avatar" src="/images/team/sidney-bell.jpg"> Sidney Bell,
		<img width=40 class="img-avatar" src="/images/team/gytis-dudas.jpg"> Gytis Dudas,
		<img width=40 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,
		<br>
		<img width=40 class="img-avatar" src="/images/team/barney-potter.jpg"> Barney Potter,
		<img width=40 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,
		<img width=40 class="img-avatar" src="/images/team/louise-moncla.jpg"> Louise Moncla
		</p>

	<p class="tiny"><b>Influenza</b>: WHO Global Influenza Surveillance Network, Richard Neher,
	Colin Russell, Andrew Rambaut, Dave Wentworth, Becky Garten, Marta &#321;uksza,
	Michael L&auml;ssig</p>

	<p class="tiny"><b>Zika</b>: Nick Loman, Nuno Faria, Oli Pybus, Josh Quick, Kristian Andersen,
	Nathan Grubaugh, Jason Ladner, Gustavo Palacios, Sharon Isern, Gytis Dudas, Alli Black,
	Barney Potter, Esther Ellis, Louise Moncla, Diana Rojas</p>

	<p class="tiny"><b>Nextstrain</b>: Richard Neher, James Hadfield, Colin Megill, Sidney Bell,
	Charlton Callender, Barney Potter, John Huddleston, Emma Hodcroft</p>

	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="16%" data-src="/images/talks/logo_fred_hutch_vertical.png">
		&nbsp; &nbsp;
		<img width="12%" data-src="/images/talks/logo_pew.png">
		&nbsp; &nbsp;
		<img width="11%" data-src="/images/talks/logo_hhmi.jpg">
		&nbsp; &nbsp;
		<img width="13%" data-src="/images/talks/logo_wellcome_trust.jpg">
		&nbsp; &nbsp;
		<img width="11%" data-src="/images/talks/logo_nih.jpg">
		&nbsp; &nbsp;
		<img width="16%" data-src="/images/talks/logo_osp.jpg">
	</div>
</section>

<section data-background="#000000">
</section>
