---
layout: paper
title: Multiplex PCR method for MinION and Illumina sequencing of Zika and other virus genomes directly from clinical samples
image: /images/papers/quick-multiplex-zika-sequencing.png
authors: Quick J, Grubaugh ND, Pullan ST, Claro IM, Smith AD, Gangavarapu K, Oliveira G, Robles-Sikisaka R, Rogers TF, Beutler NA, Burton DR, Lewis-Ximenez LL, Goes de Jesus J, Giovanetti M, Hill S, Black A, Bedford T, Carroll MW, Nunes M, Alcantara LCJ, Sabino EC, Baylis SA, Faria N, Loose M, Simpson JT, Pybus OG, Andersen KG, Loman NJ
year: 2017
ref: Quick et al. 2017. Nat Protoc.
journal: "Nat Protoc. 12: 1261-1276."
doi: 10.1038/nprot.2017.066
pdf: /pdfs/papers/quick-multiplex-zika-sequencing.pdf
---

# Abstract

Genome sequencing has become a powerful tool for studying emerging infectious diseases; however, genome sequencing directly from clinical samples without isolation remains challenging for viruses such as Zika, where metagenomic sequencing methods may generate insufficient numbers of viral reads. Here we present a protocol for generating coding-sequence complete genomes comprising an online primer design tool, a novel multiplex PCR enrichment protocol, optimised library preparation methods for the portable MinION sequencer (Oxford Nanopore Technologies) and the Illumina range of instruments, and a bioinformatics pipeline for generating consensus sequences. The MinION protocol does not require an internet connection for analysis, making it suitable for field applications with limited connectivity. Our method relies on multiplex PCR for targeted enrichment of viral genomes from samples containing as few as 50 genome copies per reaction. Viral consensus sequences can be achieved starting with clinical samples in 1-2 days following a simple laboratory workflow. This method has been successfully used by several groups studying Zika virus evolution and is facilitating an understanding of the spread of the virus in the Americas.
