---
layout: post
title: Molecular evolution of the human B cell repertoire
author: Trevor Bedford
link: /papers/mccoy-bcell-evol/
image: /images/blog/bcell_overview.png
---

Another paper posted to the [arXiv](http://arxiv.org/abs/1403.3066)... this one on a new and exciting research direction for me, looking at sequence changes within an individual's B cell repertoire as an evolutionary process, amenable to modern phylogenetic inference methods.  Basically, in order to promote immunity that protects against a wide range of possible pathogens, the adaptive immune response creates [antibodies](http://en.wikipedia.org/wiki/Antibody) that target specific [antigens](http://en.wikipedia.org/wiki/Antigen).  Antibodies are encoded in the immunoglobulin genes of [B cells](http://en.wikipedia.org/wiki/B_cell).  Rather than attempting to encode all possible antibodies within the human genome (which would be impossible), B cell immunoglobulin sequences are seeded by a process of random rearrangement of germline (inherited) sequences, termed [VDJ recombination](http://en.wikipedia.org/wiki/VDJ_recombination).  After this initial rearrangement, B cells that successfully (if partially) bind to antigen are stimulated to reproduce, during which time additional mutations are induced through [somatic hypermutation](http://en.wikipedia.org/wiki/Somatic_hypermutation).  Mutated B cells that bind better will outcompete poorer binders via Darwinian selection resulting in B cell [affinity maturation](http://en.wikipedia.org/wiki/Affinity_maturation) towards circulating antigens.  A schematic of this whole process can be seen [here](/images/blog/bcell_overview_detail.png).

It's been exciting starting to look at B cell affinity maturation from an evolutionary perspective.  [Erick Matsen](http://matsen.fhcrc.org/) has really inspired this interest and it's been great helping him, [Connor McCoy](http://cmccoy.github.io/) and [Vladimir Minin](http://www.stat.washington.edu/vminin/) develop this project.  [In this paper](/papers/mccoy-bcell-evol/), we fit substitution models and analyzed selective pressures in a short-read deep-sequencing data set of B cell receptor genes from three healthy individuals (courtesy of [Harlan Robins](http://www.fhcrc.org/en/labs/profiles/robins-harlan.html)).  We found strikingly similar patterns across individuals in terms of exactly which sites evolved and which nucleotides replacements occurred.  In this case, we did a pairwise analysis comparing observed read to matched germline sequence.  Future methodological development will focus on identifying B cells that descend from a common rearrangement event and reconstructing their phylogenies.