---
layout: talk_reveal3
title: Seasonal influenza reassortment and circulation patterns
venue: Fred Hutch
event: Comp Bio Faculty Seminar
author: Trevor Bedford
draft: true
---

<section data-background="#000">
	<h1 class="title">{{ page.title }}</h1>
	<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
</section>

<section data-background="#000">
	<h1 class="title">Influenza evolution</h1>
</section>

<section>
	<h3>Population turnover is extremely rapid</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_beast.png">
</section>

<section>
	<h3>Clades emerge, die out and take over</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_clades.png">
</section>

<section>
	<h3>Clades show rapid turnover</h3>
	<img class="stretch" data-src="/images/talks/prediction_combined_trajectories.png">
</section>

<section>
	<h3>Dynamics driven by antigenic drift</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree.png">
</section>

<section>
	<h3>Drift necessitates vaccine updates</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_vaccines.png">
</section>

<section>
	<h3>H3N2 vaccine updates occur every ~2 years</h3>
	<img class="stretch" data-src="/images/talks/flu_vaccine_updates_2018.png">
</section>

<section>
	<h3>Vaccine strain selection by WHO</h3>
	<img class="stretch" data-src="/images/talks/flu_vaccine_schedule.jpg">
</section>

<section>
	<h3>Up-to-date analysis publicly available at:</h3>
	<h2>
		<a href="http://nextstrain.org/flu">
			<span class="c1s">n</span><span class="c2s">e</span><span class="c3s">x</span><span class="c4s">t</span><span class="c5s">s</span><span class="c6s">t</span><span class="c7s">r</span><span class="c8s">a</span><span class="c9s">i</span><span class="c10s">n</span><span class="text-gray">.org/flu</span>
		</a>
	</h2>
</section>

<!-- Demo focusing on the emergence and spread of 3c2a and 3c3a viruses -->

<section>
	<h3>Current H3N2 diversity</h3>
	<img class="stretch" data-src="/images/talks/nextflu_feb2018_h3n2_tree_clades.png">
</section>

<section>
	<h3>Current H3N2 diversity</h3>
	<img class="stretch" data-src="/images/talks/nextflu_feb2018_h3n2_timetree_clades.png">
</section>

<section>
	<h3>Two clades have been growing rapidly</h3>
	<img class="stretch" data-src="/images/talks/nextflu_feb2018_h3n2_tree_lbi.png">
</section>

<section>
	<h3>Clade A2 more recently increasing</h3>
	<img class="stretch" data-src="/images/talks/nextflu_feb2018_h3n2_freq_clades_regions.png">
</section>

<section>
	<h3>Reassortment event appears to drive success of clade A2</h3>
	<img class="stretch" data-src="/images/talks/nextflu_feb2018_h3n2_ha_na_reassortment.png">
</section>

<section>
	<h3>Reassortment event appears to drive success of clade A2</h3>
	<img class="stretch" data-src="/images/talks/nextflu_feb2018_h3n2_freq_clade_A3re.png">
</section>

<section>
	<h3>Tangletrees to look at reassortment</h3>
</section>

<section>
	<h3>Like to do something more systematic here</h3>
</section>

<!---------------------------------->

<section data-background="#000">
	<h1 class="title">Geographic circulation</h1>
</section>

<section>
	<h3>Geographic circulation patterns drive evolutionary outcomes</h3>
</section>

<section>
	<h3>Clades commonly show region-specific circulation patterns</h3>
	<img class="stretch" data-src="/images/talks/nextflu_feb2018_h3n2_freq_clades_regions.png">
</section>

<section>
	<h3>Phylogeny of H3 with geographic history</h3>
	<img class="stretch" src="/images/talks/pathogen_phylogeny_flu.png">
	<div class="citation">
		<a href="http://bedford.io/papers/bedford-global-circulation/">Bedford et al. 2015. Nature.</a>
	</div>
</section>

<section>
	<h3>Infer geographic transition matrix</h3>
	<img class="stretch" src="/images/talks/flu_geo_h3_map.png">
	<div class="citation">
		<a href="http://bedford.io/papers/bedford-global-circulation/">Bedford et al. 2015. Nature.</a>
	</div>
</section>

<section>
	<h3>Geographic location of phylogeny trunk</h3>
	<img class="stretch" src="/images/talks/flu_geo_h3_trunk.png">
	<div class="citation">
		<a href="http://bedford.io/papers/bedford-global-circulation/">Bedford et al. 2015. Nature.</a>
	</div>
</section>

<section>
	<h3>Region-specific ancestry</h3>
	<img class="stretch" src="/images/talks/flu_geo_h3_hist.png">
	<div class="citation">
		<a href="http://bedford.io/papers/bedford-global-circulation/">Bedford et al. 2015. Nature.</a>
	</div>
</section>

<section>
	<h2>Which regions are most predictive of future outcomes?</h2>
</section>

<section>
	<h3>Correlation between clade frequencies</h3>
	<img class="stretch" src="/images/talks/flu_geo_corr_scatterplot.png">
</section>

<section>
	<h3>Correlations across regions</h3>
	<img class="stretch" src="/images/talks/flu_geo_corr_matrix_0m_lag.png">
</section>

<section>
	<h3>Correlations across regions with 6-month lag</h3>
	<img class="stretch" src="/images/talks/flu_geo_corr_matrix_6m_lag.png">
</section>

<section>
	<h3>Correlations across regions with 12-month lag</h3>
	<img class="stretch" src="/images/talks/flu_geo_corr_matrix_12m_lag.png">
</section>

<section>
	<h3>Frequency diffusion model</h3>
	<p>Single deme</p>
	<p>$$X(t+\mathrm{d}t) = \mathcal{N}\big( X(t) , \; \mathrm{d}t \, \sigma^2 \, X(t) \, (1-X(t)) \big),$$</p>
	<p>Multiple deme</p>
	<p>$$X(t+\mathrm{d}t) = \mathcal{N}\big( X(t) + \mathrm{d}t \, \alpha \, (Y(t)-X(t)), \; \mathrm{d}t \, \sigma^2 \, X(t) \, (1-X(t)) \big),$$</p>
	<p>$$Y(t+\mathrm{d}t) = \mathcal{N}\big( Y(t) + \mathrm{d}t \, \beta \, (X(t)-Y(t)), \; \mathrm{d}t \, \sigma^2 \, Y(t) \, (1-Y(t)) \big),$$</p>
	<p>where $\alpha$ represents flow into deme X and $\beta$ represents flow into deme Y</p>
</section>

<section>
	<h3>Migration pulls frequencies to be similar to each other</h3>
	<img class="stretch" src="/images/talks/flu_geo_diffusion_model_equal_streamplot.png">
</section>

<section>
	<h3>Biased migration pulls deme Y more strongly than deme X</h3>
	<img class="stretch" src="/images/talks/flu_geo_diffusion_model_biased_streamplot.png">
</section>

<section>
	<h3>Simulate from this biased diffusion model</h3>
	<img class="stretch" src="/images/talks/flu_geo_diffusion_model_biased_trajplot.png">
</section>

<section>
	<h3>This shows asymmetry in lagged correlation</h3>
	<img class="stretch" src="/images/talks/flu_geo_diffusion_model_biased_corrplot.png">
</section>

<!----------------------------------------->

<section data-background="#000">
	<h1 class="title">Forecasting</h1>
</section>

<section>
	<h3>Fitness models can project clade frequencies</h3>
	<br>
	<p>Clade frequencies $X$ derive from the fitnesses $f$ and frequencies $x$ of constituent viruses, such that</p>
	<p>$$\hat{X}_v(t+\Delta t) = \sum_{i:v} x_i(t) \, \mathrm{exp}(f_i \, \Delta t)$$</p>
	<p>This captures clonal interference between competing lineages</p>
	<img class="stretch" data-src="/images/talks/clonal_interferance.jpg">
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig 2014</a>
	</div>
</section>

<section>
	<h3>Fit viruses have:</h3>
	<br>
	<ol>
		<li class="larger">Amino acid changes at epitope sites</li>
		<li class="larger">Antigenic novelty based on HI</li>
		<li class="larger">Rapid phylogenetic growth</li>
	</ol>
</section>

<section>
	<h3>We predict fitness based on a simple formula</h3>
	<br>
	<p>where the fitness $f$ of virus $i$ is estimated as</p>
	<p>$$\hat{f}_i = \beta^\mathrm{HI} \, f_i^\mathrm{HI} + \beta^\mathrm{freq} \, f_i^\mathrm{freq}$$</p>
	<p>where $f_i^\mathrm{HI}$ measures antigenic drift via HI and $f_i^\mathrm{freq}$ measures clade growth/decline</p>
	<p>We learn coefficients and validate model based on previous 15 H3N2 seasons</p>
</section>

<section>
	<h3>Clade growth rate is well predicted (<i>&rho;</i> = 0.66)</h3>
	<img class="stretch" data-src="/images/talks/prediction_validation_growth_rates.png">
</section>

<section>
	<h3>Growth vs decline correct in 84% of cases</h3>
	<img class="stretch" data-src="/images/talks/prediction_validation_growth_rates_window.png">
</section>

<section data-transition="slide-in zoom-out" data-transition-speed="slow">
	<h3>Trajectories show more detailed congruence</h3>
	<img class="stretch" data-src="/images/talks/prediction_trajectories_grid.png">
</section>

<section data-transition="zoom-in slide-out">
	<h3>Trajectories show more detailed congruence</h3>
	<img class="stretch" data-src="/images/talks/prediction_trajectories_zoom.png">
</section>

<section>
	<h3>Current forecast favors drifted A1b clade</h3>
	<img class="stretch" data-src="/images/talks/nextflu_feb2018_h3n2_freq_prediction.png">
</section>

<section>
	<h3>Plan to do full genome fitnesses. Predict HA clade growth, but incorporate NA fitness effects.</h3>
	<h3>Also incorporate geography.</h3>
</section>

<!----------------------------------------->

<section data-background="#000">
	<img width=100 src="/images/talks/giphy_virus.gif">
</section>
